Composer/packages/client/src/pages/botProject/SkillBotExternalService.tsx

```diff
@@ -197,19 +207,34 @@ export const SkillBotExternalService: React.FC<SkillBotExternalServiceProps> = (
         />
         <div ref={luisKeyFieldRef}>
           <TextFieldWithCustomButton
-            ariaLabel={formatMessage('LUIS key')}
-            buttonText={formatMessage('Use custom LUIS key')}
-            errorMessage={!rootLuisKey ? formatMessage('Root Bot LUIS key is empty') : ''}
-            id={'luisKey'}
-            label={formatMessage('LUIS key')}
-            placeholder={formatMessage('Enter LUIS key')}
+            ariaLabel={formatMessage('LUIS authoring key')}
+            buttonText={formatMessage('Use custom LUIS authoring key')}
+            errorMessage={!rootLuisKey ? formatMessage('Root Bot LUIS authoring key is empty') : ''}
+            id={'luisAuthoringKey'}
+            label={formatMessage('LUIS authoring key')}
+            placeholder={formatMessage('Enter LUIS authoringkey')}
             placeholderOnDisable={rootLuisKey}
             required={isLUISKeyNeeded}
             value={skillLuisKey}
             onBlur={handleLUISKeyOnBlur}
             onRenderLabel={onRenderLabel}
           />
         </div>
+        <div ref={luisKeyFieldRef}>
+          <TextFieldWithCustomButton
+            ariaLabel={formatMessage('LUIS endpoint key')}
+            buttonText={formatMessage('Use custom LUIS endpoint key')}
+            errorMessage={!rootLuisEndpointKey ? formatMessage('Root Bot LUIS endpoint key is empty') : ''}
+            id={'luisEndpointKey'}
+            label={formatMessage('LUIS endpoint key')}
+            placeholder={formatMessage('Enter LUIS endpoint key')}
+            placeholderOnDisable={rootLuisEndpointKey}
+            required={isLUISKeyNeeded}
```

Comment:
This key should be not be a required key. Authoring keys are the key we usually have. I think this key is only to increase LUIS deployment limits

Reply:
Thanks for catching that.